profile-widget/profilewidget2.cpp

```diff
@@ -575,14 +575,7 @@ void ProfileWidget2::plotDive(const struct dive *d, bool force, bool doClearPict
 	if (currentState == EMPTY)
 		setProfileState();
 
-	// next get the dive computer structure - if there are no samples
-	// let's create a fake profile that's somewhat reasonable for the
-	// data that we have
 	struct divecomputer *currentdc = select_dc(&displayed_dive);
-	Q_ASSERT(currentdc);
-	if (!currentdc || !currentdc->samples)
-		fake_dc(currentdc);
```

Comment:
would it make sense to print out a `qWarning()` if there aren't any samples?
I'm sure you checked all the paths, but still... or am I too paranoid?